Box.V2/Models/BoxSignTemplate.cs

```diff
@@ -0,0 +1,128 @@
+        public virtual BoxSignTemplateCustomBranding CustomBranding { get; private set; }
+
+        /// <summary>
+        /// Set the number of days after which the created signature request will automatically expire if not completed.
+        /// By default, we do not apply any expiration date on signature requests, and the signature request does not expire.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldDaysValid)]
+        public virtual int? DaysValid { get; private set; }
+
+        /// <summary>
+        /// Message to include in signature request email. The field is cleaned through sanitization of specific characters.
+        /// However, some html tags are allowed. Links included in the message are also converted to hyperlinks in the email.
+        /// The message may contain the following html tags including a, abbr, acronym, b, blockquote, code, em, i, ul, li, ol, and strong.
+        /// Be aware that when the text to html ratio is too high, the email may end up in spam filters. Custom styles on these tags are not allowed.
+        /// If this field is not passed, a default message will be used.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldEmailMessage)]
+        public virtual string EmailMessage { get; private set; }
+
+        /// <summary>
+        /// Subject of signature request email. This is cleaned by signature request. If this field is not passed, a default subject will be used.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldEmailSubject)]
+        public virtual string EmailSubject { get; private set; }
+
+        /// <summary>
+        /// Name of the template.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldName)]
+        public virtual string Name { get; private set; }
+
+        /// <summary>
+        /// The destination folder to place final, signed document and signing log.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldParentFolder)]
+        public virtual BoxFolder ParentFolder { get; private set; }
+
+        /// <summary>
+        /// Box's ready-sign link feature enables you to create a link to a signature request that you've created from a template.
+        /// Use this link when you want to post a signature request on a public form — such as an email, social media post,
+        /// or web page — without knowing who the signers will be.
+        /// Note: The ready-sign link feature is limited to Enterprise Plus customers and not available to Box Verified Enterprises.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldReadySignLink)]
+        public virtual BoxSignTemplateReadySignLink ReadySignLink { get; private set; }
+
+        /// <summary>
+        /// List of signers for the template.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldSigners)]
+        public virtual List<BoxSignRequestSigner> Signers { get; private set; }
```

Comment:
In the open-api spce signers are defined there as "inline" schema - https://developer.box.com/reference/resources/sign-template/#param-signers that differs from already existing BoxSignRequestSinger object implemented based on this schema - https://developer.box.com/reference/resources/sign-request/#param-signers-inputs. You can for example look at different possible values in inputs.type. Is it problem with open-api? It looks like these schemas should be shared as they are very similar and probably data is hydrated from the same service.

If it's not a problem with open api and they are in fact two separate schemas they should be different classes in the SDK too.